api/controllers/Depmu_EntryController.js

```diff
@@ -0,0 +1,94 @@
+/* global Prebooking DepmuEntryPrebookingValidatorService Movement */
+'use strict';
+
+var ValidationError = require('../lib/exceptions/ValidationError');
+var moment = require('moment-timezone');
+moment.tz.setDefault("Europe/London");
```

Comment:
this should be done somewhere more globally, we also probably want to clarify expected behaviour around daylight savings with the business